src/components/molecules/NavList.tsx

```diff
@@ -1,53 +1,107 @@
-import React from 'react';
-import {List, ListItemButton, ListItemText, ListSubheader} from '@mui/material';
+import { List, ListItemButton, ListItemText, ListSubheader } from '@mui/material';
+import { Link, useLocation } from 'react-router-dom';
+
+const commonStyle = {
+  fontWeight: 'bold',
+  bgcolor: 'black',
+  color: 'white',
+  fontSize: 20,
+  textAlign: 'right',
+};
+
+function generateButtonStyle(isSelected: boolean, link: string) {
+  return {
+    ...(isSelected && {
+      backgroundColor: 'lightgrey',
+      padding: '1px 16px',
+      borderRadius: '24px 0 0 24px',
+      '&:hover': {
+        backgroundColor: 'lightgrey',
+      },
+    }),
+    color: isSelected ? 'black' : 'white',
+  };
+}
+
+export function NavList() {
+  const location = useLocation();
+
+  const isLinkSelected = (link: string) => location.pathname === link;
 
-function NavList (){
   return (
     <List
-      sx={{ width: '100%', maxWidth: 360, bgcolor: 'background.paper' }}
+      sx={{ width: '100%', height: '100%', maxWidth: 360, ...commonStyle }}
       component="nav"
     >
-      {/* TODO: routing on click and show current selection*/}
-      <ListItemButton>
+      <ListItemButton
+        component={Link}
+        to="/"
+        sx={{ ...generateButtonStyle(isLinkSelected('/'), '/'), textAlign: 'right' }}
+      >
         <ListItemText primary="Recent Work" />
       </ListItemButton>
-      <ListItemButton>
-        <ListItemText primary="About" />
-      </ListItemButton>
-      <ListItemButton>
+      <ListItemButton
+        component={Link}
+        to="/contact"
+        sx={{ ...generateButtonStyle(isLinkSelected('/contact'), '/contact'), textAlign: 'right' }}
+      >
         <ListItemText primary="Contact" />
       </ListItemButton>
-      
+      <ListItemButton
+        component={Link}
+        to="/about"
+        sx={{ ...generateButtonStyle(isLinkSelected('/about'), '/about'), textAlign: 'right' }}
+      >
+        <ListItemText primary="About" />
+      </ListItemButton>
+
       {/* WIP style sub items */}
-      <List component="div" >
-        <ListSubheader component="div">
-          ROLES
+      <List component="div">
+        <ListSubheader component="div" sx={{ ...commonStyle }}>
+          ROLE
         </ListSubheader>
-        <ListItemButton>
+        <ListItemButton
+          component={Link}
+          to="/director-producer"
```

Comment:
I think @ChuckBTaylor made a comment about these routes in another PR. We're going to put a filter on them.

Reply:
Yes!
I think I overlooked since it was talked about on another PR. 
I'll address later today.

Reply:
Ok, changes made to add filter. 
Now reads like this: `/video-list?filter=director-producer`

Clicking on one of the NavList items returns this URL:
`http://localhost:3000/video-list?filter=documentary`